schemas/documents/shared/gettingStartedResourcesCard.js

```diff
@@ -0,0 +1,43 @@
+export default {
```

Comment:
set this as a document so content editors can reuse a card within different "Resources for getting started" sections